<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="inputFormula">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>461</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>461</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Unesite formulu (promenljive imenovati sa p1, p2, .., pi)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prove">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Dokazi!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>471</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="notR">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>~R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="notL">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>~L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="andL">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>^L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="andR">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>^R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orL">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>v L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orR">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>v R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="impL">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>=&gt; L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="impR">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>=&gt; R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orR_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>T L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orR_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>F R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ass">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ass</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="preostali">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>451</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>461</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Preostalo za dokazivanje:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="weakL">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>weakL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="weakR">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>430</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>weakR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1188</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
